docs/en/observability/cloud-monitoring/aws/ingest-aws-options.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[ingest-aws-options]]
+==  Ingest options
+
+++++
+<titleabbrev>Ingest options</titleabbrev>
+++++
+
+You have a number of options for ingesting data with AWS. The following table helps you identify which option best fits your needs:
+
+[cols="1,1,1,1,1",options="header"]
+|===
+|
+|Firehose
+|ESF
+|Elastic Agent
+|Beats
+
+| *logs*
+| {yes-icon}
+| {yes-icon}
+| {yes-icon}
+| {yes-icon} 
+
+| *metrics*
+| {no-icon}
+| {no-icon}
+| {yes-icon}
+| {yes-icon} 
+
+| *PrivateLink support / VPC*
+| {no-icon}
+| {yes-icon}
+| {yes-icon}
+| {yes-icon} 
+
+| *pros*
+| Managed service, no management overhead, supports scaling
+| Supports scaling, built-in support for SQS
+| Supports all integrations, manages multiple agents using Fleet
+| Large configuration options 
+
+| *cons*
+| Few configuration options
+| Partial integrations support
+| Not a managed service, requires management and monitoring, no scaling
+| Not a managed service, requires management and monitoring, no scaling 
+
+|===
```

Comment:
Firelens is missing from this table. I'm guessing because it's complementary to these ingestion options. It'd be good to explain that below the table.

Reply:
As discussed and agreed with KatrinF [here](https://github.com/elastic/observability-docs/issues/1672#issuecomment-2020553011), Firlens will be added in a follow-up rework.